src/devices/directv.c

```diff
@@ -0,0 +1,532 @@
+/** @file
+    DirecTV RC66RX Remote Control decoder.
+
+    Copyright (C) 2019 Karl Lohner <klohner@thespill.com>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+ */
+
+/**
+DirecTV RC66RX Remote Control decoder.
+
+The device uses PCM encoding with 600 μs for each raw signal bit.
+
+A signal transmission contains a variable number of rows separated by
+approximately 27,600μs of gap.
+
+Each row is a SYNC packet (either a Long SYNC at 15 bits or a Short SYNC at
+10 bits), 20 Data Packets (2, 3, or 4 bits each, deliniated by space to mark bit
+transitions), and an End Of Row (EOR) packet (4 bits).
+
+Full row is between 54 and 99 bits, but maybe typically around 65 bits (39,000μs).
+
+E.g. rtl_433 -R 0 -X '-X n=DirecTV,m=FSK_PCM,s=600,l=600,g=30000,r=80000'
+
+Data Row layout:
+
+SYNC + DATA + EOR (repeated at least once, but could continue indefinitely as
+long as button is held down)
+
+SYNC is either 000111111111100 (first row in signal) or 0001111100
+(subsequent rows in transmission)
+
+DATA (20 data packets) and EOR (1 data packet) is converted to bits or EOR
+marker as such:
+
+| Raw Data Packet Bits | Decoded Logical Data Bits |
```

Comment:
I think the proper name is symbol. You transmit symbols that translate to bits or special exceptions.

Reply:
Right.  I've updated the description to use more consistent and appropriate terms.  Does it make more sense now?